scripts/generate_tutorial_audio_cosyvoice.py

```
@@ -0,0 +1,200 @@
+    model = CosyVoice(model_path)
+    print(f"[CosyVoice] 模型加载完成，声色: {SPEAKER}，语速: {SPEED}")
+
+    ok = skip = fail = 0
+    total = len(steps)
+    project_root = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+    for i, (name, text) in enumerate(steps, 1):
+        out_path = os.path.join(project_root, out_dir, f"{name}.wav")
+        if os.path.exists(out_path) and not force:
+            print(f"[{i:02d}/{total}] 跳过（已存在）: {name}.wav")
+            skip += 1; ok += 1
```

Ruff (0.15.2)

[error] Multiple statements on one line (semicolon)

(E702)